### DISCUSSION TITLE
Is It Possible To Get A Cells Height Or Width?

### DISCUSSION THREAD

**Discussion by MarkWildey** [qax-os/excelize#1627] [General]
Currently, we have 

- func (f *File) GetColWidth(sheet, col string) (float64, error)
- func (f *File) GetRowHeight(sheet string, row int) (float64, error)

but is there a way to calculate an individual cell width or height?

I need to iterate over a column and identify the cell height, i.e. some cells in the column will be merged

Thanks in advance

**Comment by xuri** [comment]
Hi @MarkWildey, please use the `GetMergeCells` function to get a merged cell, and then you can know the merged cell range (left-top cell coordinates and right-bottom cell coordinates), and covert the cell coordinates to row/column number by the `CellNameToCoordinates` function, accumulate the columns width and rows height in this range by the `GetColWidth` and `GetRowHeight` functions, you will get the merged cell's height and width.
